assets/src/edit-story/app/prepublish/guidance/test/media.js

```diff
@@ -150,5 +150,82 @@ describe('Pre-publish checklist - media guidelines (guidance)', () => {
     expect(result.elementId).toStrictEqual(tooLongVideo.id);
   });
 
+  it('should return a message if the video element is larger than 1080x1920 and not optimized', () => {
+    const largeUnoptimizedVideo = {
+      id: 202,
+      type: 'video',
+      resource: {
+        isOptimized: false,
+        sizes: {
+          full: {
+            height: 2160,
+            width: 3840,
+          },
+        },
+      },
+    };
+
+    const result = mediaGuidance.videoElementOptimized(largeUnoptimizedVideo);
+    expect(result).not.toBeUndefined();
+    expect(result.message).toBe('Video not optimized');
+    expect(result.type).toStrictEqual('guidance');
+    expect(result.elementId).toStrictEqual(largeUnoptimizedVideo.id);
```

Comment:
Nit: this test doesn't need to check if result is undefined, but not a big deal
```suggestion
    expect(result.message).toBe('Video not optimized');
    expect(result.type).toStrictEqual('guidance');
    expect(result.elementId).toStrictEqual(largeUnoptimizedVideo.id);
```

Reply:
lol yea I think I was just pulling from tests above. does seem a lil redundant now that you point it out. can update